src/compare.jl

```diff
@@ -0,0 +1,59 @@
+#  Copyright 2015, Vincent Leclere, Francois Pacaud and Henri Gerard
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#############################################################################
+#  Compare the optimal values and control returned by different instances
+#  of SDDP on the same problem
+#############################################################################
+
+"""
+Compare different sets of parameters to solve an instance of SDDP
+
+Parameters:
+- model (SPmodel)
+    the stochastic problem we want to benchmark
+
+- SDDParametersCollection ()
+
+- seeds (Int)
+    The seed which is chosen by the users to compare parameters on the same aleas
+
+- scenarios ()
```

Comment:
@frapac done the for the documentation
@leclere and @frapac, I have done the change